ios/CameraView+RecordVideo.swift

```diff
@@ -6,41 +6,17 @@
 //  Copyright © 2020 mrousavy. All rights reserved.
 //
 
-import AVFoundation
-
-enum OculaExamMode: String {
-    case normal = "normal"
-    case enhancedRecording = "enhancedRecording"
-    case enhancedMetric = "enhancedMetric"
-    case enhancedAll = "enhancedAll"
+struct TorchControl {
+    var torchLength: Double
+    var torchDeday: Double
```

Comment:
```suggestion
    var torchDelay: Double
```

Reply:
it fixed.
I installed spellcheck extension to VSCode. Hope this fault does not happen.